arch/x86/soc/intel_quark/quark_se/soc.h

```diff
@@ -89,6 +90,10 @@
 
 #endif /*  _ASMLANGUAGE */
 
+void usb_dw_clock_disable(void);
+int usb_dw_clock_enable(void);
+void _usb_dw_int_unmask(void);
```

Comment:
remove the underscore from _usb_dw_int_unmask, this is no longer internal.

Reply:
done